test/ex_ach/field_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule ExAch.FieldTest do
+  use ExUnit.Case
+
+  defmodule DummyField do
+    use ExAch.Field, default_content: 5, specifications: [type: :integer, length: 1]
+  end
+
+  setup_all do
+    :dummy_field
```

Comment:
I think we could remove this line by extracting the field_name logic into a `field_name` function in `Field`, and then add a test here: `assert Field.field_name(dummy) == :dummy_field`.